src/UI/UIUtils.java

```diff
@@ -274,6 +274,14 @@ public static COSpriteSpec instance(Commander co)
       return spec;
     }
 
+    @Override
+    public String toString()
+    {
+      if( paletteNames.containsKey(color) )
+        return getCanonicalFactionName(this);
+      return color.toString() + " " + faction.name;
```

Comment:
Only get the canonical name if the color is in `paletteNames`? Aren't they all?

Reply:
`MISC` is not.

Reply:
Right, the infamous "No CO".